src/test/java/org/jabref/logic/search/query/SearchQueryLuceneConversionTest.java

```diff
@@ -10,64 +12,55 @@
 
-                // String with a space
-                Arguments.of("title:image\\ processing AND author:smith", "title = \"image processing\" AND author= smith"),
+                Arguments.of("MatchNoDocsQuery(\"\")", "title = image"),
+                Arguments.of("content:\"imag process\" annotations:\"imag process\"", "\"image processing\" AND author = smith"),
+                Arguments.of("+(content:imag annotations:imag) +(content:process annotations:process)", "image AND (title = term OR processing)"),
+                Arguments.of("(content:imag annotations:imag) (content:process annotations:process)", "image OR (title = term OR processing)"),
+                Arguments.of("MatchNoDocsQuery(\"\")", "title = \"image processing\" AND author = smith"),
 
-                // We renamed fields
-                Arguments.of("any:somecontent", "anyfield = somecontent"),
-                Arguments.of("keywords:somekeyword", "anykeyword = somekeyword"),
-                Arguments.of("citationkey:somebibtexkey", "key = somebibtexkey")
-        );
-    }
+                Arguments.of("content:neighbou?r annotations:neighbou?r", "neighbou?r"),
+                Arguments.of("content:neighbo* annotations:neighbo*", "neighbo*"),
+                Arguments.of("MatchNoDocsQuery(\"\")", "title = neighbou?r"),
+                Arguments.of("MatchNoDocsQuery(\"\")", "(title == chocolate) OR (author == smith)"),
 
-    @ParameterizedTest
-    @MethodSource
-    void transformationNormal(String expected, String query) {
-        String result = SearchQueryConversion.searchToLucene(query);
-        assertEquals(expected, result);
-    }
-
-    public static Stream<Arguments> transformationRegularExpression() {
-        return Stream.of(
-                Arguments.of("any:* AND -groups:/.+/", "groups != .+"),
-                Arguments.of("( any:* AND -groups:/.+/ ) AND ( any:* AND -readstatus:/.+/ )", "groups != .+ and readstatus != .+"),
-                Arguments.of("author:/(John|Doe).+(John|Doe)/", "author = \"(John|Doe).+(John|Doe)\""),
-                Arguments.of("any:/rev*/", "anyfield=rev*"),
-                Arguments.of("any:/*rev*/", "anyfield=*rev*")
+                Arguments.of("content:/(John|Doe).+(John|Doe)/ annotations:/(John|Doe).+(John|Doe)/", "any =~ \"(John|Doe).+(John|Doe)\""),
+                Arguments.of("content:/rev*/ annotations:/rev*/", "anyfield=~ rev*"),
+                Arguments.of("content:/*rev*/ annotations:/*rev*/", "anyfield=~ *rev*"),
+                Arguments.of("(-content:/.+/) (-annotations:/.+/)", "any !=~ .+"),
+                Arguments.of("(-content:/.+/) (-annotations:/.+/)", "groups !=~ .+ AND any !=~ .+")
         );
     }
 
     @ParameterizedTest
     @MethodSource
-    void transformationRegularExpression(String expected, String query) {
-        String result = SearchQueryConversion.searchToLucene(query);
+    void testSearchConversion(String expected, String searchExpression) {
```

Comment:
No test names should begin with `test`. - https://docs.openrewrite.org/recipes/java/testing/cleanup/removetestprefix -- I am not sure why OpenRewrite didn't catch this.

Reply:
`testSearchConversion` begin with `test` 😅